docs/api/pq.rst

```diff
@@ -149,7 +149,7 @@ Enumerations
     after a connection has been established. Other statuses might only be seen
     during the connection phase and are considered internal.
 
-    .. seealso:: :pq:`PQstatus()` returns this value.
+    .. seealso:: :pq:`PQstatus()` and `PQcancelStatus()` return this value.
```

Comment:
I did in another commit not pushed here because we'd need to change `libpq_docs_version = "devel"` for the docs to build; if that's okay for you (considering this would affect published docs I think by linking to https://www.postgresql.org/docs/devel/), I can add the commit here.